cypress/component/Auth/auth.spec.ts

```diff
@@ -1,29 +1,47 @@
 /* eslint-disable no-undef */
 
+import {OidcBroker} from '../../../src/libs/auth/oidcBroker';
 import {Auth} from '../../../src/libs/auth/auth';
-import {Config} from '../../../src/libs/config';
-import {GoogleIS} from '../../../src/libs/googleIS';
 import {OAuth2} from '../../../src/libs/ajax/OAuth2';
 import {Storage} from '../../../src/libs/storage';
-import { v4 as uuid } from 'uuid';
+import {v4 as uuid} from 'uuid';
+import {mockOidcUser} from './mockOidcUser';
 
-describe('Auth', function () {
+describe('Auth Failure', function () {
+  it('Sign In error throws expected message', async function () {
+    cy.stub(OidcBroker, 'signIn').returns(null);
+    cy.on('fail', (err) => {
+      return err.message !== Auth.signInError();
+    });
+    Auth.signIn();
```

Comment:
This function still seems to be async, does it need to be `await`ed?

Reply:
Good call, but `await`-ing breaks the test, I think I need to use `then`